[16.0][MIG] web_m2x_options_manager : Migrated to 16.0

QuocDuong1306 opened the pull request.

kutyel committed:
[MIG] web_widget_image_download

yajo committed:
[MIG] web_widget_image_download: Update for v9.

smurf-U committed:
[10.0][MIG]migrate web_widget_image_download

oca-transbot committed:
OCA Transbot updated translations from Transifex

ernestotejeda committed:
[MIG] web_widget_image_download: Migration to 11.0

pedrobaeza committed:
[IMP] README by fragments + manifest

OCA-git-bot committed:
[UPD] README.rst

oca-travis committed:
[UPD] Update web_widget_image_download.pot

tarteo committed:
[IMP] web_disable_export_group: Tests

ernestotejeda committed:
[MIG] web_widget_image_download: Migration to 12.0

oca-travis committed:
[UPD] Update web_widget_image_download.pot

oca-transbot committed:
Update translation files

Updated by Update PO files to match POT (msgmerge) hook in Weblate.

chienandalu committed:
[ADD] web_widget_domain_editor_dialog: New module

Tardo committed:
[FIX] Tests

darshan-odoo committed:
[MIG] web_widget_domain_editor_dialog: Migration to 12.0.

OCA-git-bot committed:
[UPD] README.rst

liweijie0812 committed:
Added translation using Weblate (Chinese (Simplified))

liweijie0812 committed:
Translated using Weblate (Chinese (Simplified))

Currently translated at 100.0% (3 of 3 strings)

Translation: web-12.0/web-12.0-web_disable_export_group
Translate-URL: https://translation.odoo-community.org/projects/web-12-0/web-12-0-web_disable_export_group/zh_CN/

liweijie0812 committed:
Added translation using Weblate (Chinese (Simplified))

liweijie0812 committed:
Added translation using Weblate (Chinese (Simplified))

liweijie0812 committed:
Translated using Weblate (Chinese (Simplified))

Currently translated at 100.0% (1 of 1 strings)

Translation: web-12.0/web-12.0-web_widget_image_download
Translate-URL: https://translation.odoo-community.org/projects/web-12-0/web-12-0-web_widget_image_download/zh_CN/

pedrocs-exo committed:
Added translation using Weblate (Portuguese)

pedrocs-exo committed:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (1 of 1 strings)

Translation: web-12.0/web-12.0-web_widget_image_download
Translate-URL: https://translation.odoo-community.org/projects/web-12-0/web-12-0-web_widget_image_download/pt/

badbole committed:
Added translation using Weblate (Croatian)

badbole committed:
Translated using Weblate (Croatian)

Currently translated at 100.0% (3 of 3 strings)

Translation: web-12.0/web-12.0-web_disable_export_group
Translate-URL: https://translation.odoo-community.org/projects/web-12-0/web-12-0-web_disable_export_group/hr/

Tardo committed:
[IMP] web_widget_image_download: black, isort

Tardo committed:
[MIG] web_widget_image_download: Migration to 13.0

oca-travis committed:
[UPD] Update web_widget_image_download.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: web-13.0/web-13.0-web_widget_image_download
Translate-URL: https://translation.odoo-community.org/projects/web-13-0/web-13-0-web_widget_image_download/

sbidoul committed:
pre-commit update: prettier xml indent

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[UPD] README.rst

claudiagn committed:
Added translation using Weblate (Spanish)

claudiagn committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (3 of 3 strings)

Translation: web-12.0/web-12.0-web_disable_export_group
Translate-URL: https://translation.odoo-community.org/projects/web-12-0/web-12-0-web_disable_export_group/es/

CarlosRoca13 committed:
[IMP] web_widget_domain_editor_dialog: black, isort, prettier

CarlosRoca13 committed:
[MIG] web_widget_domain_editor_dialog: Migration to v13.0

pedrocs-exo committed:
Added translation using Weblate (Portuguese)

pedrocs-exo committed:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (3 of 3 strings)

Translation: web-12.0/web-12.0-web_disable_export_group
Translate-URL: https://translation.odoo-community.org/projects/web-12-0/web-12-0-web_disable_export_group/pt/

joao-p-marques committed:
[IMP] web_disable_export_group: black, isort, prettier

joao-p-marques committed:
[MIG] web_disable_export_group: Migration to 13.0

TT26186

Kev-Roche committed:
[MIG] web_disable_export_group: Migration to 14.0

JordiBForgeFlow committed:
[13.0][web_widget_domain_editor_dialog][FIX] Error when used in wizards.
When clicking edit the list of record an error happened due to lack
of proper initialization of the default domain.

claudiagn committed:
Added translation using Weblate (Spanish)

claudiagn committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (1 of 1 strings)

Translation: web-13.0/web-13.0-web_widget_image_download
Translate-URL: https://translation.odoo-community.org/projects/web-13-0/web-13-0-web_widget_image_download/es/

newtratip committed:
[13.0][FIX] web_widget_domain_editor_dialog: Example picture not display on README

Tardo committed:
[IMP] web_disable_export_group: Disable 'export all' button

Peerapong-Supasompob committed:
[IMP] web_widget_image_download: black, isort, prettier

Peerapong-Supasompob committed:
[14.0][MIG] web_widget_image_download

SilvioC2C committed:
[14.0] web_m2x_options_manager: New module to manage create/edit options from interface

hkapatel-initos committed:
[IMP] web_widget_domain_editor_dialog: black, isort, prettier

hkapatel-initos committed:
[MIG] web_widget_domain_editor_dialog: Migration to 14.0

oca-travis committed:
[UPD] Update web_m2x_options_manager.pot

OCA-git-bot committed:
[UPD] README.rst

yankinmax committed:
[UPD] web_m2x_options_manager: Add option to prevent Create/Edit wizard to pop up

oca-travis committed:
[UPD] Update web_m2x_options_manager.pot

OCA-git-bot committed:
web_m2x_options_manager 14.0.1.1.0

Tardo committed:
[ADD] web_widget_one2many_tree_line_duplicate

etobella committed:
[IMP] web_disable_export_group: Differenciate between the XLSX export and the standard export

Commit:
[MIG] web_m2x_options_manager: Migration to 15.0

- Fix postprocess inherit (removed from v15), and use _postprocess_tag_field
for create/edit options: In v14 it worked with inheriting the "postprocess"
method because in each recursive call it passed a section of the view, in the
way that it went through all the child nodes of the root node, and passed through
the fields, and now in v15 the method "_postprocess_view "it just does the while,
and we never go through the nodes.

- Fix pylint errors: xml-deprecated-tree-attributte, attribute-string-redundant,
translation-positional-used

Commit:
[UPD] Update web_m2x_options_manager.pot

OCA-git-bot committed:
[UPD] README.rst

oca-travis committed:
[UPD] Update web_widget_image_download.pot

OCA-git-bot committed:
[UPD] README.rst

chienandalu committed:
[IMP/FIX] web_widget_domain_editor_dialog: work in dialog

- IMP: Now it's possible to work with the domain editor when the widget
has de in_dialog option.
- FIX: When a boolean opertator was present in the domain, the editor dialog
would raise an error as the default filter couldn't be interpretated.

TT32827

celm1990 committed:
[MIG] web_widget_domain_editor_dialog: Migration to 15.0

alessandro-fiorino committed:
Added translation using Weblate (Italian)

alessandro-fiorino committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (1 of 1 strings)

Translation: web-14.0/web-14.0-web_widget_image_download
Translate-URL: https://translation.odoo-community.org/projects/web-14-0/web-14-0-web_widget_image_download/it/

Tardo committed:
[IMP] web_widget_one2many_tree_line_duplicate: Clone all fields

manuelregidor committed:
[MIG]web_widget_image_download: Migration to 15.0

Commit:
[UPD] Update web_widget_image_download.pot

OCA-git-bot committed:
[UPD] README.rst

chienandalu committed:
[FIX] web_widget_one2many_tree_line_duplicate: clone wisely

- When we clone a line, we can be interested in preloading those fields
availiable in other field views available. Otherwise, we could be unable
to get the proper states in the cloned line.
- When we load the cloned field values from the rpc method, we must
discard those fields not available in the views or some subproccess
could try to get their state causing errors.

TT38191

chienandalu committed:
[FIX] web_widget_one2many_tree_line_duplicate: cloned record relations

On the cloned record we were unable to change the values of the
relational fields we we just had edited it. Here is solved adding a
pseudo onchange method that triggers the proper refreshed values as if
the record was made manually.

TT38191

pedrobaeza committed:
[OU-FIX] web_disable_export_group: Proper migration scripts

The previous adaptation was not correct due to:

- The old XML-ID is already missing on post stage.
- There's now only one group, so no need of all the imply + group
  assignation stuff like in previous version.

etobella committed:
[OU] web_disable_export_group: Fix when migration happened on 13.0

AmmarOfficewalaSerpentCS committed:
[MIG] web_m2x_options_manager : Migration to 16.0.

chienandalu committed:
[MIG] web_disable_export_group: Migration to 15.0

TT43531

baimont committed:
[ADD] web_field_tooltip

baimont committed:
[IMP] web_field_tooltip - one rpc call

benwillig committed:
[IMP] display a helper to add a tooltip on a field. Display the tooltip on list views. One tooltip per field and do not handle user specific tooltips

dzungtran89 commented:
@QuocDuong1306 The function does not work
- I tried to set up as below image

  ![image](https://github.com/QuocDuong1306/web/assets/46039081/cfb0d5b2-e308-4e71-b724-3cafd9fc1d0a)

- But the setup field still allows to create objects
  
  ![image](https://github.com/QuocDuong1306/web/assets/46039081/9733b2fd-ebe0-43aa-99d9-001904098041)

benwillig committed:
[REF] use FormViewDialog instead of do_action, allow to unlink a tooltip from the helper, splitted resources into multiple files

QuocDuong1306 commented:
Hi @dzungtran89 , I updated

dzungtran89 reviewed (approved):
lgtm

OCA-git-bot committed:
[UPD] README.rst

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (1 of 1 strings)

Translation: web-15.0/web-15.0-web_widget_image_download
Translate-URL: https://translation.odoo-community.org/projects/web-15-0/web-15-0-web_widget_image_download/it/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: web-15.0/web-15.0-web_widget_image_download
Translate-URL: https://translation.odoo-community.org/projects/web-15-0/web-15-0-web_widget_image_download/

chienandalu committed:
[FIX] web_widget_domain_editor_dialog: keep context

The dialog needs to keep the context so translations and other things
work as expected.

TT45543

CarlosRoca13 committed:
[IMP] web_widget_domain_editor_dialog: pre-commit stuff--no-verify

CarlosRoca13 committed:
[MIG] web_widget_domain_editor_dialog: Migration to 16.0

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (4 of 4 strings)

Translation: web-15.0/web-15.0-web_disable_export_group
Translate-URL: https://translation.odoo-community.org/projects/web-15-0/web-15-0-web_disable_export_group/es/

Commit:
[UPD] Update web_field_tooltip.pot

OCA-git-bot committed:
[BOT] post-merge updates

houzefa-abba committed:
[FIX] web_timeline: Wrap fields_get arg into a list

By contract the first argument of the `fields_get` method is supposed to
be a list. Before this fix, `web_timeline` would call `fields_get` with
a string instead of a list.

Fortunately in case only 1 field is being grouped, this worked as Odoo
does an `x in y` comparison in its `fields_get` implementation, which
does pass in a simple `"project_id" in "project_id"` case.

But that call remains invalid and can break when `fields_get` has been
tweaked by other modules.

cuongnmtm committed:
[IMP] web_remember_tree_column_width: adjust SCSS selector

Previously, only columns with the class o_column_sortable had the custom
SCSS applied to them. As a result, columns like HTML were not sortable
and didn't benefit from the custom SCSS.

This commit made changes to the SCSS selector to include all table's
columns in the list view. However, the record selector (the first
checkbox column) and the column optional (the last column) that have the
class o_list_controller were excluded to avoid unexpected behavior.

FranzPoize committed:
ADD web_hide_field_with_key module

OCA-git-bot committed:
Merge PR #2758 into 16.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update web_widget_domain_editor_dialog.pot

OCA-git-bot committed:
[BOT] post-merge updates

chienandalu committed:
[MIG] web_disable_export_group: Migration to 16.0

TT45881

OCA-git-bot committed:
Merge PR #2759 into 16.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update web_disable_export_group.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: web-16.0/web-16.0-web_disable_export_group
Translate-URL: https://translation.odoo-community.org/projects/web-16-0/web-16-0-web_disable_export_group/

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (1 of 1 strings)

Translation: web-16.0/web-16.0-web_send_message_popup
Translate-URL: https://translation.odoo-community.org/projects/web-16-0/web-16-0-web_send_message_popup/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (19 of 19 strings)

Translation: web-16.0/web-16.0-web_pivot_computed_measure
Translate-URL: https://translation.odoo-community.org/projects/web-16-0/web-16-0-web_pivot_computed_measure/it/

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (2 of 2 strings)

Translation: web-16.0/web-16.0-web_widget_domain_editor_dialog
Translate-URL: https://translation.odoo-community.org/projects/web-16-0/web-16-0-web_widget_domain_editor_dialog/it/

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (3 of 3 strings)

Translation: web-16.0/web-16.0-web_disable_export_group
Translate-URL: https://translation.odoo-community.org/projects/web-16-0/web-16-0-web_disable_export_group/it/

OCA-git-bot committed:
Merge PR #2458 into 16.0

Signed-off-by dreispt

Commit:
[UPD] Update web_copy_confirm.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #2349 into 16.0

Signed-off-by dreispt

Commit:
[UPD] Update web_widget_image_webcam.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: web-16.0/web-16.0-web_widget_image_webcam
Translate-URL: https://translation.odoo-community.org/projects/web-16-0/web-16-0-web_widget_image_webcam/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: web-16.0/web-16.0-web_copy_confirm
Translate-URL: https://translation.odoo-community.org/projects/web-16-0/web-16-0-web_copy_confirm/

OCA-git-bot committed:
Merge PR #2734 into 16.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (6 of 6 strings)

Translation: web-16.0/web-16.0-web_widget_image_webcam
Translate-URL: https://translation.odoo-community.org/projects/web-16-0/web-16-0-web_widget_image_webcam/it/

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (2 of 2 strings)

Translation: web-16.0/web-16.0-web_copy_confirm
Translate-URL: https://translation.odoo-community.org/projects/web-16-0/web-16-0-web_copy_confirm/it/

houzefa-abba committed:
[IMP] web_timeline: Overflow text, add hover colors

When an item label does not fit in its date-range box, overflow
according to
https://visjs.github.io/vis-timeline/examples/timeline/items/rangeOverflowItem.html

Previous CSS code was already trying to do that, but was selecting
`.vis-item.vis-item-content` instead of `.vis-item .vis-item-content`.

Displaying overflow text brings up layout issues solved by removing the
forced-100% width instruction.

This change also adds highlight when hovering a box, which is useful on
text that has overflown (as it has no borders).

houzefa-abba committed:
[IMP] web_timeline: Add demo view on cron tasks

This way we can showcase this module without project_timeline.

OCA-git-bot committed:
Merge PR #2752 into 16.0

Signed-off-by sebastienbeau

Commit:
[UPD] Update web_hide_field_with_key.pot

OCA-git-bot committed:
[BOT] post-merge updates

benwillig committed:
[MIG] web_field_tooltip: Migration to 16.0

CarlosRoca13 committed:
[IMP] web_refresher: Backport refresh way from v17

OCA-git-bot committed:
Merge PR #2598 into 16.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update web_notify.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: web-16.0/web-16.0-web_notify
Translate-URL: https://translation.odoo-community.org/projects/web-16-0/web-16-0-web_notify/

hbrunn committed:
[FIX] web_tree_duplicate: adapt readme to behavior

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (19 of 19 strings)

Translation: web-16.0/web-16.0-web_notify
Translate-URL: https://translation.odoo-community.org/projects/web-16-0/web-16-0-web_notify/it/

OCA-git-bot committed:
Merge PR #2713 into 16.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #2776 into 16.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates

houzefa-abba committed:
[IMP] web_timeline: Align to top instead of bottom

This avoids somewhat ugly very large "Unassigned" first row.

OCA-git-bot committed:
Merge PR #2788 into 16.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates

chienandalu committed:
[FIX] web_responsive: fix status widget visibility

Before this change the status widget was hidden as it had no width. You
can check it installing the project module and going to a task. Now you
can see the status widget along the task name.

OCA-git-bot committed:
Merge PR #2792 into 16.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #2775 into 16.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates

ibuioli committed:
Translated using Weblate (Spanish (Argentina))

Currently translated at 100.0% (20 of 20 strings)

Translation: web-16.0/web-16.0-web_pwa_oca
Translate-URL: https://translation.odoo-community.org/projects/web-16-0/web-16-0-web_pwa_oca/es_AR/

CarlosRoca13 committed:
[IMP] web_refresher: Be able to refresh from report views

CarlosRoca13 committed:
[IMP] web_widget_one2many_tree_line_duplicate: pre-commit stuff

OCA-git-bot committed:
Merge PR #2783 into 16.0

Signed-off-by dreispt

Commit:
[UPD] Update web_field_tooltip.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #2511 into 16.0

Signed-off-by dreispt

Commit:
[UPD] Update web_ir_actions_close_wizard_refresh_view.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: web-16.0/web-16.0-web_field_tooltip
Translate-URL: https://translation.odoo-community.org/projects/web-16-0/web-16-0-web_field_tooltip/

CarlosRoca13 committed:
[MIG] web_widget_one2many_tree_line_duplicate: Migration to 16.0

CarlosRoca13 committed:
[IMP] web_responsive: Set viewer width to 100% to better use the space

Before this changes, when previewing a pdf there was a lot of lost space

Now, it is filling all the space

OCA-git-bot committed:
Merge PR #2797 into 16.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #2787 into 16.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update web_refresher.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: web-16.0/web-16.0-web_refresher
Translate-URL: https://translation.odoo-community.org/projects/web-16-0/web-16-0-web_refresher/

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (2 of 2 strings)

Translation: web-16.0/web-16.0-web_refresher
Translate-URL: https://translation.odoo-community.org/projects/web-16-0/web-16-0-web_refresher/it/

Commit:
[UPD] Update web_group_expand.pot

Commit:
[UPD] Update web_pivot_computed_measure.pot

Commit:
[UPD] Update web_widget_image_webcam.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: web-16.0/web-16.0-web_pivot_computed_measure
Translate-URL: https://translation.odoo-community.org/projects/web-16-0/web-16-0-web_pivot_computed_measure/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (27 of 27 strings)

Translation: web-16.0/web-16.0-web_pivot_computed_measure
Translate-URL: https://translation.odoo-community.org/projects/web-16-0/web-16-0-web_pivot_computed_measure/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (17 of 17 strings)

Translation: web-16.0/web-16.0-web_ir_actions_close_wizard_refresh_view
Translate-URL: https://translation.odoo-community.org/projects/web-16-0/web-16-0-web_ir_actions_close_wizard_refresh_view/it/

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (28 of 28 strings)

Translation: web-16.0/web-16.0-web_field_tooltip
Translate-URL: https://translation.odoo-community.org/projects/web-16-0/web-16-0-web_field_tooltip/it/

CarlosRoca13 committed:
[IMP] web_refresher: Charge t-inherit-mode extension views just after the main

By doing this, the extension view is charged just after the main, so
if some other views are using this other, to make a primary one,
like project is doing for project.task control_panel, the refresher
will continue be shown.

hbrunn committed:
[ADD] web_widget_pattern

OCA-git-bot committed:
Merge PR #2804 into 16.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates

sbidoul committed:
use codecov action v4

OCA-git-bot committed:
[IMP] update dotfiles

OCA-git-bot committed:
Merge PR #2720 into 16.0

Signed-off-by luisg123v

OCA-git-bot committed:
[BOT] post-merge updates

glitchov committed:
Translated using Weblate (French)

Currently translated at 82.1% (23 of 28 strings)

Translation: web-16.0/web-16.0-web_field_tooltip
Translate-URL: https://translation.odoo-community.org/projects/web-16-0/web-16-0-web_field_tooltip/fr/

vincent-hatakeyama committed:
[FIX] web_m2x_options: fix usage errors

glitchov committed:
[FIX] web_field_tooltip: ir.model and ir.model.fields access if not admin

OCA-git-bot committed:
Merge PR #2806 into 16.0

Signed-off-by dreispt

Commit:
[UPD] Update web_widget_pattern.pot

Commit:
[UPD] Update web_widget_pattern_partner_autocomplete.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #2823 into 16.0

Signed-off-by dreispt

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (6 of 6 strings)

Translation: web-16.0/web-16.0-web_widget_open_tab
Translate-URL: https://translation.odoo-community.org/projects/web-16-0/web-16-0-web_widget_open_tab/it/

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (1 of 1 strings)

Translation: web-16.0/web-16.0-web_widget_pattern
Translate-URL: https://translation.odoo-community.org/projects/web-16-0/web-16-0-web_widget_pattern/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (1 of 1 strings)

Translation: web-16.0/web-16.0-web_widget_pattern_partner_autocomplete
Translate-URL: https://translation.odoo-community.org/projects/web-16-0/web-16-0-web_widget_pattern_partner_autocomplete/it/

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (19 of 19 strings)

Translation: web-16.0/web-16.0-web_notify
Translate-URL: https://translation.odoo-community.org/projects/web-16-0/web-16-0-web_notify/pt_BR/

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (20 of 20 strings)

Translation: web-16.0/web-16.0-web_pwa_oca
Translate-URL: https://translation.odoo-community.org/projects/web-16-0/web-16-0-web_pwa_oca/pt_BR/

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (2 of 2 strings)

Translation: web-16.0/web-16.0-web_refresher
Translate-URL: https://translation.odoo-community.org/projects/web-16-0/web-16-0-web_refresher/pt_BR/

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (1 of 1 strings)

Translation: web-16.0/web-16.0-web_widget_x2many_2d_matrix
Translate-URL: https://translation.odoo-community.org/projects/web-16-0/web-16-0-web_widget_x2many_2d_matrix/pt_BR/

OCA-git-bot committed:
Merge PR #2790 into 16.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (6 of 6 strings)

Translation: web-16.0/web-16.0-web_widget_open_tab
Translate-URL: https://translation.odoo-community.org/projects/web-16-0/web-16-0-web_widget_open_tab/pt_BR/

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (4 of 4 strings)

Translation: web-16.0/web-16.0-web_advanced_search
Translate-URL: https://translation.odoo-community.org/projects/web-16-0/web-16-0-web_advanced_search/pt_BR/

pilarvargas-tecnativa committed:
[FIX] web_responsive: Remove unnecessary media query

Remove unnecessary media query as odoo has support
for long titles. This style causes an error in the
layout of the affected views.

OCA-git-bot committed:
Merge PR #2832 into 16.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #2824 into 16.0

Signed-off-by dreispt

OCA-git-bot committed:
[BOT] post-merge updates

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (15 of 15 strings)

Translation: web-16.0/web-16.0-web_m2x_options
Translate-URL: https://translation.odoo-community.org/projects/web-16-0/web-16-0-web_m2x_options/pt_BR/

CarlosRoca13 committed:
[FIX] web_dialog_size: Fixes on legacy Dialogs

The problems found on Legacy Dialogs are the next:

1. The modal can't be dragged
2. The button for extending and reduce the modal is doing nothing
3. The x for close the modal is showed twice

You can test it with dialogs showed on Vault module

OCA-git-bot committed:
Merge PR #2843 into 16.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #2809 into 16.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update web_widget_one2many_tree_line_duplicate.pot

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (1 of 1 strings)

Translation: web-16.0/web-16.0-web_widget_one2many_tree_line_duplicate
Translate-URL: https://translation.odoo-community.org/projects/web-16-0/web-16-0-web_widget_one2many_tree_line_duplicate/it/

pilarvargas-tecnativa committed:
[FIX] web_responsive: Remove view that does not allow zooming in mobile view

This feature is already defined for backend views in odoo.
Views can be consulted at https://github.com/odoo/odoo/blob/54f84aedf8da14f4f3298febc44c4fec103e3a47/addons/web/views/webclient_templates.xml#L294

TT49728

OCA-git-bot committed:
Merge PR #2862 into 16.0

Signed-off-by legalsylvain

OCA-git-bot committed:
[BOT] post-merge updates

TesaLazarenko committed:
[FIX] web_refresher: new record refresh

OCA-git-bot committed:
Merge PR #2889 into 16.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates

CarlosRoca13 committed:
[FIX] web_refresher: Define displayRefresher in setup instead of using a getter

Before this changes whe trying to access to a view with the refresher,
in mobile view, an error was thrown.

As it is not necessary to define the value in a get function, defining
the value in the setup solves the problem.

OCA-git-bot committed:
Merge PR #2891 into 16.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates

CarlosRoca13 committed:
[IMP] web_responsive: Set AttachmentViewer z-index upper than chatter

Before this changes when oppening an attachment from the chatter the
attachment was behind the chatter and if we try to hide it, the
attachement disappears.

With this changes the AttachmentViewer appears in front of the chatter
and the problem does not exists anymore.

OCA-git-bot committed:
Merge PR #2894 into 16.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates

manuelregidor committed:
[IMP] web_widget_image_download: black, isort, prettier

manuelregidor committed:
[MIG]web_widget_image_download: Migration to 16.0

carolinafernandez-tecnativa committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (18 of 18 strings)

Translation: web-16.0/web-16.0-web_responsive
Translate-URL: https://translation.odoo-community.org/projects/web-16-0/web-16-0-web_responsive/es/

CarlosRoca13 committed:
[IMP] web_responsive: Change replace for less dangerous extension options

Replacing all the template is a bit dangerous if any other module tries
to extend that tamplate.

So this changes make the same as it is done on before, but using the
other options provided by xpath that are less dangerous.

OCA-git-bot committed:
Merge PR #2899 into 16.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update web_responsive.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: web-16.0/web-16.0-web_responsive
Translate-URL: https://translation.odoo-community.org/projects/web-16-0/web-16-0-web_responsive/

CarlosRoca13 committed:
[FIX] web_company_color: Set defaults hardcoded to avoid errors charging the assetbundle

When trying to charge the assetbundle the theme-color("primary") function
is not found what causes problems when trying to set the default values.

With this changes the default value is the value set on https://github.com/odoo/odoo/blob/ad9f8fae717aae4cad5b3715c1f73ee906b64f8d/addons/web/static/src/scss/primary_variables.scss#L55
this change avoid the problems and the module works fine.

Addapted too the class for required fields.

OCA-git-bot committed:
Merge PR #2897 into 16.0

Signed-off-by etobella

Commit:
[UPD] Update web_widget_image_download.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: web-16.0/web-16.0-web_widget_image_download
Translate-URL: https://translation.odoo-community.org/projects/web-16-0/web-16-0-web_widget_image_download/

OCA-git-bot committed:
Merge PR #2902 into 16.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates
